### ISSUE TITLE
Question : Must --simulate option should simulate database pulling (mysqldump) ?

### ISSUE THREAD

**Issue by eldk** [welaika/wordmove#626]
Hello,

Must --simulate option should simulate database pulling ?

```
wordmove pull --all -v --debug --simulate

>f..t...... wp-content/languages/themes/twentytwentyone-fr_FR.po
sent 853 bytes  received 6,625 bytes  4,985.33 bytes/sec
total size is 5,594,481  speedup is 748.13 (DRY RUN)

▬▬ Pulling Database ▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬
```
Nothing happened ?

The config use ssh + rsync + mysqldump.

Thanks,

Eric

**Comment by pioneerskies** [comment]
Hi,

actually when simulating the pull action will solely print database actions, actually skipping them. Sincerly I've never realized my own expectation about what simulating the DB transfer/adapt could mean...

As it stands it shows to you only the operation that would have been ran, without running it.

If you have different expectations I'd like to discuss them and eventually track them down in a feature request.

Thanks for the enquiry.

**Comment by eldk** [comment]
Thanks for your answer.

>As it stands it shows to you only the operation that would have been ran, without running it.

So I should see log lines under -- Pulling Database --- ?

Those lines should reflect what should be done without --simulate option ?

I have nothing under (not a single line)  -- Pulling Database --- no database actions at all.

Thanks,

Eric

PS : I was using wordmove with sftp before switching to SSH, and, if I remember database operation were logged under -- Pulling Database when --simulate option is used (even if nothing is really done).

**Comment by pioneerskies** [comment]
Yes, sorry: the "operation" from the wordmove pov is "Pulling database".

This is what i get with FTP

<img width="314" alt="immagine" src="https://user-images.githubusercontent.com/904690/148223637-3bf6521d-e472-42f2-b675-c1b3f476caa1.png">

While I'm working on the new wordmove version I'm compelling about what to log and what to not log while simulating. This is an example from the alpha version

<img width="746" alt="immagine" src="https://user-images.githubusercontent.com/904690/148223819-8c6d59c1-d4cf-49f6-99e1-27a896c05ca6.png">

But you didn't express any expectation about what you think should be useful. That would help with future implementation. :)
